packages/modules/order/src/migrations/Migration20250522181137.ts

```diff
@@ -0,0 +1,21 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20250522181137 extends Migration {
+  override async up(): Promise<void> {
+    this.addSql(
+      `DELETE FROM "order_summary" WHERE "order_id" NOT IN (SELECT id FROM "order");`
```

Comment:
question: Is this a safe operation? Seems a bit drastic to delete all orphaned order summaries, but perhaps, they aren't used so it's more a clean-up that anything? @carlos-r-l-rodrigues

Reply:
Yes, this is a cleanup of summaries of already deleted orders. Since cascade wasn't properly set up, they weren't deleted. We also cannot add the foreign key constraint if we don't remove summaries without their order first.

Reply:
Shouldn't we only query orders that are not deleted in the `IN(...)` clause?

Reply:
Okay, I think I partly thought this was the intention. 

I can't see how summaries could exist with an order ID, that isn't in the DB, since we are not hard-deleting any entities, but I guess it's fine to include the script just in case.